src/diff_util.rs

```diff
@@ -0,0 +1,641 @@
+// Copyright 2020 The Jujutsu Authors
```

Comment:
Should it be 2022? I don't know much about copyright law. Maybe you used 2020 since you're just moving code?

Reply:
It's just copy paste from commands.rs. Updated to 2022, but should it be 2020-2022?

Reply:
Good question. Looking at https://opensource.google/documentation/reference/copyright, 2020-2022 is my best guess, yes. (I just happened to find that page via a Google search; I don't think the advice there is specific to Google-originated projects.)

Reply:
Fixed to 2020-2022, thanks!